app/android/build.gradle

```diff
@@ -55,3 +55,19 @@ allprojects {
         maven { url 'https://www.jitpack.io' }
     }
 }
+
+// fix detox with auth0 error, reference: https://github.com/auth0/react-native-auth0/issues/387#issuecomment-918483390 
+subprojects {
+    afterEvaluate { subproject ->
+        if (subproject.name == 'react-native-auth0') {
+            def String taskRequests = getGradle().getStartParameter().getTaskRequests().toString()
+            if (taskRequests.contains("assembleAndroidTest")) {
+                android {
+                    defaultConfig {
+                        manifestPlaceholders = [auth0Domain: "E2E_TEST", auth0Scheme: "com.E2E.TEST"]
+                    }
+                }
+            }
+        }
```

Comment:
Will any of these values change if another user sets up their own app?

Reply:
Can you add a comment right above this code block explaining why you had to do this? And also add the link to https://github.com/auth0/react-native-auth0/issues/387#issuecomment-918483390

Customers might wonder what this is for.

Reply:
@charliesantos yes I did, already on line 59? Or did you want to change the text?

Reply:
Sorry missed that. +1